io.elementary.code.json

```diff
@@ -0,0 +1,247 @@
+{
+	"app-id": "io.elementary.code",
+	"runtime": "org.gnome.Platform",
+	"runtime-version": "3.24",
+	"sdk": "org.gnome.Sdk",
+	"command": "scratch-text-editor",
+	"rename-desktop-file": "org.pantheon.scratch.desktop",
+	"rename-appdata-file": "org.pantheon.scratch.appdata.xml",
+	"finish-args": [
+		"--filesystem=host",
+		/* Wayland and X11 access */ 
+		"--socket=wayland",
+		"--socket=x11",
+		/* own dbus name */ 
+		"--own-name=org.pantheon.scratch",
+		"--own-name=org.pantheon.terminal",
+		"--own-name=org.gnome.zeitgeist",
```

Comment:
terminal or zeitgeist?

https://github.com/elementary/scratch/blob/76afb93f2d7ed7f9c0ed8efb7c7ad238d0f737a4/plugins/terminal/terminal.vala#L222

IIRC, there were errors without the zeitgeist bus name.